modules/claims_api/app/sidekiq/claims_api/evidence_waiver_builder_job.rb

```diff
@@ -11,21 +12,26 @@ class EvidenceWaiverBuilderJob < ClaimsApi::ServiceBase
     #
     # @param evidence_waiver_id [String] Unique identifier of the submitted EWS
     def perform(evidence_waiver_id)
-      evidence_waiver_submission = ClaimsApi::EvidenceWaiverSubmission.find(evidence_waiver_id)
-      auth_headers = evidence_waiver_submission.auth_headers
+      lighthouse_claim = ClaimsApi::EvidenceWaiverSubmission.find(evidence_waiver_id)
+      auth_headers = lighthouse_claim.auth_headers
       output_path = ClaimsApi::EvidenceWaiver.new(auth_headers:).construct
 
-      upload_to_vbms(evidence_waiver_submission, output_path)
+      # upload to BD
+      benefits_doc_api.upload(claim: lighthouse_claim, pdf_path: output_path,
+                              doc_type: 'L705')
```

Comment:
Currently I am getting this back from this line when submitting locally (using Tamara Ellis as the test  user)
```
Unmapped VA900 (Backend Response: { status: 400, message: {:errors=>[{:detail=>"No user with the given corporate file number was found", :status=>400, :title=>"Bad Request", 
```

Reply:
@rockwellwindsor-va I was able to successfully submit with Tamara.

Reply:
Yep same now